relation_engine_server/utils/pull_spec.py

```diff
@@ -62,10 +62,20 @@ def download_specs(
     return update_name
 
 
+def _glob_specs(spec_type):
```

Comment:
Consolidated the globbing JSON/YAML files. Some of the views were actually being skipped for some reason because it was in the root directory, not in `root/**/*.ext`